### PR TITLE
Clean up unused code

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**
## Summary

- Cleaning up a lot of unused files
- Remove the export keyword from types that aren't used anywhere else

### CODE REVIEW

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: src/components/charts/index.tsx [hunk 1]
```diff
@@ -2,4 +2,3 @@ import dynamic from 'next/dynamic';
 
 export const AreaChart = dynamic(() => import('~/components/areaChart'));
 export const BarChart = dynamic(() => import('~/components/barChart'));
-export const LineChart = dynamic(() => import('~/components/lineChart'));
```
Comment (VWSCoronaDashboard4) [inline src/components/charts/index.tsx:5]: Why is this removed?

**Review by VWSCoronaDashboard4 — APPROVED** [review]

File: src/components/charts/index.tsx (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply src/components/charts/index.tsx:5]: Because it's not used
